src/commitToBranch.ts

```diff
@@ -13,7 +13,7 @@ mutation myCreateCommitOnBranch($input: CreateCommitOnBranchInput!) {
 `
 
 const commitToBranch = async ({ owner, repo }: InputFields, username: string, branchNodeId: string, branchSha: string, documentBody: string): Promise<{ ref: { id: string } }> => {
-    const now = (new Date()).toISOString();
+    const today = (new Date()).toISOString().replace(/T.*$/, "");
```

Comment:
I wonder if we should just shorten this to MM/DD/YYYY

Reply:
Something like: `new Date().toLocaleDateString(undefined, 'MM-DD-YYYY')`

Reply:
Thanks for the suggestions! I see a couple potential disadvantages:
1. `toLocaleDateString` will give different values on different [runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners), based on their location.
2. `MM/DD/YYYY` (or `MM-DD-YYYY`) results in an odd sort order, placing January 2022 reports next to January 2023 reports, rather than next to February 2022 reports.

I did consider alternative approaches, such as:
```ts
const date = new Date();
const year = date.getUTCFullYear();
const month = String(date.getUTCMonth()+1).padStart(2, '0');
const date = String(date.getUTCDate()).padStart(2, '0');
const today = `${year}-${month}-${date}`
```

But this PRs’s approach seemed simpler to me. `T` (in the regular expression) is something of a [magic string](https://en.wikipedia.org/wiki/Magic_string), which I’d generally avoid, except that in this case it’s specified in ISO 8601.

Reply:
> results in an odd sort order
@smockle good catch!